index.html

```diff
@@ -0,0 +1,328 @@
+                    <span>Boyu Capital, +4</span>
+                </a>
+            </div>
+            <div class="last-single-info">
+                <span>SoftBank-backed travel platform Klook raises $200M amid COVID-19 </span>
+                <span class="dark-grey">on TechCrunch</span>
+            </div>
+        </div>
+        <div id="additional-single-info" class="container-single-lf grid-padding">
+            <a href="#"><img src="Images/latest-founding-4.jpeg" alt="latest found venture"></a>
+            <a href="#"><h3>Wolt</h3></a>
+            <a href="" class="info-single-lf">
+                <span class="first-single-info">$530</span> <br>
+                <span class="second-single-info">VENTURE CAPITAL</span>
+            </a>
+            <div class="single-lef-intereactions flex">
+                <a href="">
+                    <i class="fas fa-ticket-alt"></i>
+                    <span>Mobile, +5</span>
+                </a>
+                <a href="#">
+                    <i class="fas fa-briefcase"></i>
+                    <span>DST Global, +9</span>
+                </a>
+            </div>
+            <div class="last-single-info">
+                <span>Food delivery startup Wolt raises $530M after tripling revenues on SiliconANGLE. </span>
+                <span class="dark-grey">on TechCrunch</span>
+            </div>
+        </div>
+    </div>
+</section>
+<!-- END latest founding -->
+
+<!-- START footer -->
+<footer class="flex-column">
+    <section class="top-footer flex-column flex">
+        <div class="socialmedia-icons flex">
+            <ul class="flex wrap">
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fab fa-facebook-square"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fab fa-youtube"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fab fa-instagram-square"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fab fa-pinterest-p"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fas fa-rss"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" rel="noopener">
+                        <i class="fas fa-envelope"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="footer-links flex">
```

Comment:
Better to change this div tag to nav-tag.

Reply:
Done